concourse-chrome-driver/terraform/deployments/670214072732/action_triggers.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+      "action": "BuildAndPushCdImage", 
+      "trigger_paths": [
+        "Dockerfile",
+	"bin"
+      ]
+  }
```

Comment:
Should this have an entry for the "BuildAndPushConcourseImage" action?

Reply:
Not entirely sure but can add if you think it's required 👍